storage/pkg/flagset/frontend.go

```diff
@@ -42,7 +42,7 @@ func FrontendWithConfig(cfg *config.Config) []cli.Flag {
 		// this can eg. be set to /eos/users
 		&cli.StringFlag{
 			Name:        "dav-files-namespace",
-			Value:       "/oc/",
+			Value:       "/users/",
```

Comment:
we no longer need to change this from /oc to eos because we onlyd change the driver of the users storage provider mounted at /users